geonode_dominode/geonode_dominode/settings.py

```diff
@@ -159,3 +159,23 @@
 
 # Add your specific LDAP configuration after this comment:
 # https://docs.geonode.org/en/master/advanced/contrib/#configuration
+
+if DEBUG:
+    LOGGING['loggers']['geonode'] = {
+        "handlers": ["console"], "level": "DEBUG",
+    }
+    LOGGING['loggers']['geonode_dominode'] = {
+        "handlers": ["console"], "level": "DEBUG"
+    }
+    LOGGING['handlers']['console']['level'] = 'DEBUG'
+
+
+CELERY_TASK_QUEUES += (
+    Queue(
+        'geonode_dominode',
+        GEONODE_EXCHANGE,
+        routing_key='geonode_dominode'),
+)
+
+CELERY_TASK_DEFAULT_QUEUE = 'default'
+CELERY_TASK_DEFAULT_ROUTING_KEY = 'default'
```

Comment:
The upstream celery settings is very broken.
It's only apparent if we uses a proper message queue backend.
For some default task in django and geonode that is not annotated, the message is lost because no task to pick them up. The settings here is just a bandaid for the default task handler. But ideally the the code in the upstream must be fixed, for example the producer/consumer implementation.